server/meca/file-generators/manifest.test.js

```diff
@@ -1,10 +1,43 @@
-const manifest = require('./manifest')
+const generateManifest = require('./manifest')
+const sampleManuscript = require('./article.test.data')
 
 describe('Manifest XML generator', () => {
-  it('returns meaningful content', async () => {
-    const xml = await manifest()
+  it('returns required items', async () => {
+    const xml = await generateManifest(sampleManuscript.files)
```

Comment:
This is no longer async.

Reply:
The actual export function is async... 
```

async function mecaExport(manuscript, content, clientIp) {
  logger.info(`Starting MECA export`, { manuscriptId: manuscript.id })
  const archive = await generate(manuscript, content, clientIp)
  await upload(archive, manuscript.id)
}
```

Reply:
That isn't the function being tested here, we're testing [`generateManuscript`](https://github.com/elifesciences/elife-xpub/blob/db80c2008a884b40bd17e4b7396d84513ceacc60/server/meca/file-generators/manifest.js#L40)

Reply:
This will be fixed in the next PR.

Reply:
see #1275